### ISSUE TITLE
[BUG] 🐛 Bug Report — Sonarr/Radarr don't detect network shares on first boot after system restart

### ISSUE THREAD

**Issue by NoisyBoyFR** [linuxserver/docker-sonarr#321]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

After a full system reboot, Sonarr and Radarr fail to detect or access the mounted network drives (e.g. /mnt/media).
When the containers first start, the paths appear missing or inaccessible.
Only after manually restarting both containers do the network shares become available and everything works normally.

Other containers (like qBittorrent, Prowlarr, and FlareSolverr) using the same mounted /mnt paths work fine immediately after reboot.

### Expected Behavior

Sonarr and Radarr should detect and access the mounted network drives on their first startup after a system reboot, without requiring a manual restart of the containers.

### Steps To Reproduce

Mount a CIFS or NFS network share to /mnt/media via /etc/fstab, e.g.:
//192.168.1.xxx/media /mnt/media cifs credentials=/root/.smbcred,iocharset=utf8,rw,vers=3.0 0 0

Deploy the following containers:
volumes:
  - /opt/stacks/media-stack1/sonarr:/config
  - /mnt:/mnt
  - 
Reboot the system.

Open Sonarr or Radarr → /mnt/media is missing or inaccessible.

Restart the container → the path is detected correctly.

### Environment

```markdown
Host OS: Debian and NAS QNAP

Docker Compose: v2.x

Images:

linuxserver/sonarr:latest

linuxserver/radarr:latest

Network mode: qnapbridge

Volumes mounted:
/opt/stacks/media-stack1/[service]/:/config
/mnt:/mnt

Network share mount:
CIFS via /etc/fstab

Other containers sharing the same mount:
qBittorrent, Prowlarr, FlareSolverr — all working fine immediately after reboot.
```

### CPU architecture

x86-64

### Docker creation

```bash
services:
  sonarr:
    image: linuxserver/sonarr:latest
    container_name: sonarr1
    entrypoint: >
      sh -c "
        /opt/stacks/media-stack1/wait-for.sh 192.168.1.42:8090 -t 300 -- echo '8090 OK' && \
        /opt/stacks/media-stack1/wait-for.sh 192.168.1.42:9696 -t 300 -- echo '9696 OK' && \
        /opt/stacks/media-stack1/wait-for.sh 192.168.1.42:8191 -t 300 -- echo '8191 OK' && \
        echo '✅ Tous les services sont prêts, attente de 60 secondes avant lancement...' && \
        sleep 60 && \
        exec /init
      "
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/Paris
      - UMASK=022
    volumes:
      - /opt/stacks/media-stack1/sonarr:/config
      - /mnt:/mnt
      - /opt/stacks/media-stack1:/opt/stacks/media-stack1
    ports:
      - 8091:8091/tcp
    network_mode: qnapbridge
    security_opt:
      - no-new-privileges:true
    restart: unless-stopped
  radarr:
    image: linuxserver/radarr:latest
    container_name: radarr1
    entrypoint: >
      sh -c "
        /opt/stacks/media-stack1/wait-for.sh 192.168.1.42:8090 -t 300 -- echo '8090 OK' && \
        /opt/stacks/media-stack1/wait-for.sh 192.168.1.42:9696 -t 300 -- echo '9696 OK' && \
        /opt/stacks/media-stack1/wait-for.sh 192.168.1.42:8191 -t 300 -- echo '8191 OK' && \
        echo '✅ Tous les services sont prêts, attente de 60 secondes avant lancement...' && \
        sleep 60 && \
        exec /init
      "
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/Paris
      - UMASK=022
    volumes:
      - /opt/stacks/media-stack1/radarr:/config
      - /mnt:/mnt
      - /opt/stacks/media-stack1:/opt/stacks/media-stack1
    ports:
      - 8092:8092/tcp
    network_mode: qnapbridge
    security_opt:
      - no-new-privileges:true
    restart: unless-stopped
networks: {}
```

### Container logs

```bash
[ls.io-init] done.
[Error] LocalizationService: Missing translation/culture resource: /app/sonarr/bin/Localization/Core/fr_FR.json 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/Lego Ninjago Masters Of Spinjitzu.S15.TRUEFRENCH.1080p.WEB.x264-FTMVHD. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/Oggy Et Les Cafards.S04.1080p.WEB.x264-FTMVHD. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/Baby Shark's Big Show.S01.TRUEFRENCH.1080p.WEB-DL.H264-FTMVHD. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/Masha et Michka S02. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/Super Wings.S04.TRUEFRENCH.1080p.WEB.x264-FTMVHD. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/Super Wings.S02.TRUEFRENCH.1080p.WEB.x264-FTMVHD. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/La.Casa.De.Papel.S01.MULTi.1080p.WEB.x264-CiELOS. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/Trolls The Beat Goes On.S03.MULTI.1080p.WEB.x264-FTMVHD. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/The New Adventures Of Winnie The Pooh.S03.TRUEFRENCH.1080p.WEB.x264-FTMVHD. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/Oggy Et Les Cafards.S06.1080p.WEB.x264-FTMVHD. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/The Powerpuff Girls.1998.RESTORED.TRUEFRENCH.1080p.WEB.x264-FTMVHD. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 
[Error] DownloadedEpisodesImportService: Import failed, path does not exist or is not accessible by Sonarr: /mnt/Partage/qbittorent/Terminé/Saison 05 -1080p Multi.x265. Ensure the path exists and the user running Sonarr has the correct permissions to access this file/folder 

................ etc ....
```

**Comment by Roxedus** [comment]
1. Modifying the entrypoint results in a unsupported configuration, support will continue once that's removed.
2. This is caused by the lan network not being present when the container start, on a systemd based distro, it's solved by having the docker service start after the networking service.
3. Due to the sleep in 1, and the sonarr and radar containers probably being the first one started, the network is present on the system when the others starts.
4. This is not a issue with the image, but rather the startup process in your nas, we don't know how to achieve the correct startup order on your OS.

**Comment by NoisyBoyFR** [comment]
I have an idea! What would be the command to tell it to reboot the docker after 15 seconds? since it works when I do it manually

**Comment by NoisyBoyFR** [comment]
I've already been offered this solution, as well as adding Requires=mnt-Partage.mount. Is this really necessary? I'll try your method and let you know how it goes.
